_includes/mode-toggle.html

```diff
@@ -142,4 +142,11 @@
 
   });
 
+  $("#mode-toggle-wrapper").keyup(function(e){
+    if(e.keyCode == 13)
+    {
```

Comment:
In order to maintain a consistent code style, please modify it as follows:

```js
$("#mode-toggle-wrapper").keyup(function(e) {
    if(e.keyCode == 13) {
    // ...
```

Reply:
Yes you are right, will do.